src/pages/Applications/List/ApplicationsList.controller.tsx

```diff
@@ -3,9 +3,20 @@ import { useApplicationsQuery } from './ApplicationsList.queries';
 import ApplicationsListView from './ApplicationsList.view';
 import EmptyState from '~/components/EmptyState';
 import Loader from '~/components/Loader';
+import { ApplicationStatus } from '~/@types/common';
+import useApplicationUpdateMutation from '~/queries/useApplicationUpdateMutation';
 
 const ApplicationsListController = (): JSX.Element => {
-  const { data, isLoading } = useApplicationsQuery();
+  const { data, isLoading, refetch } = useApplicationsQuery();
+  const { mutate: updateApplication } = useApplicationUpdateMutation({
+    onSuccess: () => {
+      refetch();
```

Comment:
Should we refetch or invalidate? When invalidating, request will be made only when necessary. When refetching, it will be done immediately even if application list is not visible (if I remember correctly)

Reply:
Ok